mod_form.php

```diff
@@ -106,9 +106,15 @@ public function definition(): void {
             )
         );
 
-        $mform->addElement('advcheckbox', 'showmaponcoursepage', get_string('showmaponcoursepage', 'learningmap'));
+        // If using learningmap course format, the map is never shown on the course page.
+        if ($this->_course->format === 'learningmap') {
+            $mform->addElement('hidden', 'showmaponcoursepage', 0);
```

Comment:
This does not seem to work. I can still see the checkbox in the form when using format_learningmap